frontend/src/metabase/query_builder/components/QuestionActions.unit.spec.tsx

```diff
@@ -0,0 +1,67 @@
+import React from "react";
+import userEvent from "@testing-library/user-event";
+import { renderWithProviders, screen } from "__support__/ui";
+import { createMockState } from "metabase-types/store/mocks";
+import { createMockEntitiesState } from "__support__/store";
+import QuestionActions from "metabase/query_builder/components/QuestionActions";
+import { createMockCard, createMockNativeCard } from "metabase-types/api/mocks";
+import { getMetadata } from "metabase/selectors/metadata";
+import Question from "metabase-lib/Question";
+
+const TEST_STRUCTURED_CARD = createMockCard();
+const TEST_NATIVE_CARD = createMockNativeCard();
+
+const iconList = [
+  { label: "bookmark icon", tooltipText: "Bookmark" },
+  { label: "info icon", tooltipText: "More info" },
+  {
+    label: "Move, archive, and more...",
+    tooltipText: "Move, archive, and more...",
+  },
+];
+
+function setup({ card = TEST_STRUCTURED_CARD } = {}) {
```

Comment:
is the default value even required if we always pass `card` in the test?
https://github.com/metabase/metabase/blob/5b2940929e46a80069a6f61c82d5e3e46e83d2ab/frontend/src/metabase/query_builder/components/QuestionActions.unit.spec.tsx#L53-L58

This gives the impression that `setup` might be called without specifying `card`.